test/core/client_channel/lb_policy/xds_override_host_test.cc

```diff
@@ -37,6 +34,28 @@ class XdsOverrideHostTest : public LoadBalancingPolicyTest {
          Json::Object{{"childPolicy", Json::Array{{child_policy_config}}}}}}});
   }
 
+  void SetUp() override {
+    grpc_core::SetEnv("GRPC_EXPERIMENTAL_XDS_ENABLE_HOST_OVERRIDE", "TRUE");
+    grpc_init();
+    policy_ = MakeLbPolicy("xds_override_host_experimental");
+  }
+
+  void TearDown() override {
+    // Note: Can't safely trigger this from inside the FakeHelper dtor,
```

Comment:
Instead of duplicating this code from the parent class, just call the method from the parent class directly:

```
LoadBalancingPolicyTest::TearDown();
```

That way, if we add anything else to the parent class `TearDown()` in the future, we won't need to duplicate it here.